boot/bootutil/include/bootutil/bootutil_log.h

```diff
@@ -28,11 +28,28 @@ extern "C" {
 
 #ifdef MCUBOOT_HAVE_LOGGING
 
+
+
+#define BOOT_LOG_DBG(...) MCUBOOT_LOG_DBG(__VA_ARGS__)
+
+#ifdef CONFIG_BOOT_SERIAL_UART_DETECT
+#define BOOT_LOG_INF(format, args...)  {\
```

Comment:
Instead of adding a BOOT_LOG_INF, which is going to affect every use of BOOT_LOG_INF, perhaps we should add a special print message call just for this uart detect. If logging is enabled, it can just use one of the logging calls, and if not, it can just `console_write` the message. It doesn't look like any of the uses need printf functionality, so this could just output simple strings.

Reply:
It's good idea, I fix it

Reply:
my NACK for what is done here